_posts/2016-2-14-Ruby-Conf-Au-2016.md

```diff
@@ -0,0 +1,65 @@
+
+TBC....
+
+## Rails Girls Next Workshop by Sarah.
+
+What you need:
+
+- Github account: [https://github.com/join?source=header-home](https://github.com/join?source=header-home)
+- Cloud 9: [https://c9.io/web/sign-up/free](https://c9.io/web/sign-up/free)
+
+The Rails Girls Next workshop is a step up from the Rails Girls workshop.
+
+The workshop started at 9am at Sea World Resort. Nothing better than a fantastic location with great morning tea.
+
+#### Why Sinatra instead of Rails?
+
+**Rails**: Things happen magically under the hood.
+
+**Sinatra**: More flexible in comparison, but would require you to implement some elements yourself.
+
+We then began learning the ways of building a Rails app from scratch through Sinatra! Develop your app on the cloud!
+
+#### [Ruby Girls Sinatra App Guide](http://railsgirls-bne.github.io/sinatra-app-guide)
+
+This was written by Sorcha and is a very excellent step-by-step guide we followed to understand how each file works in Rails. It's very easy to follow and quite self-explanatory.
+
+**Caution**: I learnt that Sinatra is NOT structured like how files in Rails would.
+
+For instance, Routing in Sinatra is controlled in the controller, whereas Routing in Rails is in a separate routes file.
+
+#### [Put Your App Online With Heroku](http://railsgirls-bne.github.io/heroku)
+
+I then deployed the app to heroku. A couple of us in the workshop were having a problem with getting the app to deploy (Error: Application not found).
+
+Here's the solution: Since Sinatra is structured slightly differently from Rails, you'll need to modify *config.ru* since gems and classes are located in *config/application.rb*
+
+![alt text](https://www.dropbox.com/sc/6bzcmtnd9v6wk90/AADdZZf7Aejm3tPKOEUSEvEpa?dl=1)
+
+Big thanks to Sorcha, Rachelle and Daphne for teaching us the tiny details of MVC and Routes.
+
+#### Bonus: Lots of freebies
+
+![alt text](https://www.dropbox.com/sc/ueg6rawwimaxw72/AADeFEa_IBJ2E-hAlcg0DQE1a?dl=1)
+
+![alt text](https://www.dropbox.com/sc/xsaziedizmf7fbn/AACGdQetrpO0ygaa0zStksjka?dl=1)
+
+**Edit**: I had initially wrote my post as "we", based on the idea of us operating as a collective unit. After discussing with Vi, I have changed all the "we/s" to "I". A great point brought up by Vi was on transparency, where we should really be giving a clear picture on what happened. In this case, only I had attended Rails Girls Next, so that's that.
+
+#### Sarah and the conference
+
+At first glance, this probably doesn't make sense.
+
+In short, I actually had to make a quick trip up to Sydney (fun fact: it's a 1.5 hour-long flight) to attend an awards ceremony. I'm humbled to have been nominated by GradConnection as one of the [Top 100 students in Australia](https://au.gradconnection.com/top100/), 2016. I believe that part of this award was related to my involvement with RGSoC, and I would like to thank Vi for being a great pair. I wouldn't have won this award without her dedication to the project. 
```

Comment:
@cyberkoi it's really thoughtful of you to mention the pair part, but because it's a blog about the conference and workshop, would you mind removing this part please? 
> I would like to thank Vi for being a great pair. I wouldn't have won this award without her dedication to the project.